staticcheck.conf

```diff
@@ -0,0 +1,10 @@
+checks = ["all", "-ST1000", "-ST1003", "-ST1016"]
+initialisms = ["ACL", "API", "ASCII", "CPU", "CSS", "DNS",
```

Comment:
Should we use the word “acronyms” for this variable name instead?

Reply:
initiaslisms is the standard name used in the library.

Reply:
Oh! Well, in that case... 😄